pom.xml

```diff
@@ -62,18 +62,15 @@
   </pluginRepositories>
 
   <properties>
-    <!--
-      explicitly specifying the latest version here because one we get from the parent POM
-      tends to lag behind a bit
-    -->
-    <maven-hpi-plugin.version>1.95</maven-hpi-plugin.version>
+    <maven-hpi-plugin.version>1.106</maven-hpi-plugin.version>
```

Comment:
Hmm, yeah. This change was not really meant to be in there as well (sloppy job of me reviewing the changes). I think it is not really necessary for this feature to work. However, I have to test this and clean up my pull request a bit.